### ISSUE TITLE
Remove duplicates

### ISSUE THREAD

**Issue by matentzn** [obophenotype/c-elegans-phenotype-ontology#21]
There are now a bunch of terms with more than one definition. 
[wbpheno-duplicates.txt](https://github.com/obophenotype/c-elegans-phenotype-ontology/files/2580097/wbpheno-duplicates.txt)

While we do [not yet have](https://github.com/ontodev/robot/issues/402) an ODK way to getting these, maybe it would be a good exercise to get rid of the ones already in there. Note that **most of them should be deleted from this src/ontology/wbphenotype_eq_do_no_edit.owl file**, but there is a chance that somehow the same class was defined in two separate patterns - unlikely though. Perhaps going through all these and deleting the wrong ones will give us a hint how good we were doing :)

**Comment by chris-grove** [comment]
@matentzn  OK, so just for the record, I'm going to edit the "do_no_edit" file (not sure if that is meant to be "do_not_edit"?) and delete the duplicated phenotype terms?

**Comment by matentzn** [comment]
do_not_edit should say "unless_you_are_chris_do_not_edit.owl". You alone should be allowed to make changes to this file. And yes: if you find duplicate definitions, just delete them from the do_not_edit file.

**Comment by chris-grove** [comment]
@matentzn 
Should we close this or leave open for other (eventual?) duplicates?

Also, the "wbpheno-duplicates.txt" file you linked above didn't seem to correspond to any terms with duplicate EQ definitions. I just went by the terms that we had created under the abnormalMorphology and abnormalProcess pattern, checking each one for presence in the no-edit file and that seemed the best way to find the terms with duplicate definitions. My point is, whatever query was used to generate the "wbpheno-duplicates.txt" file may not have worked as intended.
